docker/nginx.conf.d/default.conf.tpl

```diff
@@ -25,6 +25,10 @@ server {
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_http_version 1.1;
 
+    # The user header is available for logging, but not returned to the client
+    proxy_hide_header X-Takahe-User;
+    proxy_hide_header X-Takahe-User-Identity;
```

Comment:
It'll be easier to visually scan logs if you change it to `X-Takahe-Identity`

Reply:
Sure, changed. In my nginx logs, I don't log the actual header name so that didn't matter as much in my case, but sounds reasonable.